src/main/resources/resources.properties

```diff
@@ -30,6 +30,10 @@ notification.FeedbackAvailableNotification.content=Feedback for {0} [{1}] is now
 notification.OpenSubmissionNotificationAction.content=Click here to open the feedback.
 notification.CourseFileError.title=Could not access course settings file
 notification.CourseFileError.content=The local file for course settings could not be accessed. Error message: {0}.
+notification.CourseVersionError.title=Plugin incompatible with course
+notification.CourseVersionOutdatedError.content=This course project cannot be loaded because your A+ Courses plugin version is outdated. Please update the A+ Courses plugin to open this course.
+notification.CourseVersionTooNewError.content=This course is no longer supported by your A+ Courses plugin version. Please contact the course staff for assistance.
+notification.CourseVersionOutdatedWarning.content=This course recommends using a newer version of the A+ Courses plugin. Update so as not to miss out on any new useful features!
```

Comment:
Perhaps we could quickly consult on these in the weekly. I'm not too good at making these up, but "update so as not to miss out on any new useful features!" is very markety :smile: